src/it/compare-ignore/modA/pom.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='UTF-8'?>
+
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one
+  or more contributor license agreements. See the NOTICE file
+  distributed with this work for additional information
+  regarding copyright ownership. The ASF licenses this file
+  to you under the Apache License, Version 2.0 (the
+  "License"); you may not use this file except in compliance
+  with the License. You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing,
+  software distributed under the License is distributed on an
+  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+  KIND, either express or implied. See the License for the
+  specific language governing permissions and limitations
+  under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.maven.plugins.it</groupId>
+    <artifactId>ignore</artifactId>
+    <version>1.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>ignore-modA</artifactId>
+  <packaging>jar</packaging>
+  <name>ignore module A</name>
+
+  <build>
+    <resources>
+      <resource>
+        <directory>src/main/resources</directory>
+        <filtering>true</filtering>
+      </resource>
+    </resources>
+
+    <plugins>
+      <plugin>
+        <groupId>@project.groupId@</groupId>
+        <artifactId>@project.artifactId@</artifactId>
+        <version>@project.version@</version>
+        <configuration>
+          <ignore>*/ignore-modA-*.jar</ignore>
```

Comment:
this should not be configured in this module's POM but in parent

Reply:
it relates to artifacts produced by this module though, isn't it better to be 'closer' to the source?

or is artifact always intended to be configured at the 'top-level' of the build?

Reply:
and TBH, I did not even expect compare goal to be configured in pom.xml: I always use this goal fully on CLI, because I compare a rebuild done on one situation with an initial build that may have been done anywhere, and the more environmental distance between the 2 builds, the better to discover unexpected discrepencies

Reply:
Yeah, it wouldn't be part of the regular build. In practice we configure it in `pluginManagement`.